ansible/files/provision_vm.py

```
@@ -108,13 +110,23 @@ def provision_vm(api, args):
     pod_selection_spec = get_vm_pod_selection_spec(api, datastore_cluster_name)
     storage_spec = get_vm_storage_spec(name, folder, pod_selection_spec, template, clone_spec, operation_type='clone')
 
-    task = clone_template_to_datastore_cluster(api, storage_spec)
-    WaitForTask(task)
+    try:
+        task = clone_template_to_datastore_cluster(api, storage_spec)
+        WaitForTask(task)
+    except (KeyboardInterrupt, SystemExit):
```

We should rethrow an exception to close the script correctly.